cmd/skaffold/app/cmd/dev.go

```diff
@@ -38,6 +38,7 @@ func NewCmdDev(out io.Writer) *cobra.Command {
 	}
 	AddRunDevFlags(cmd)
 	AddDevFlags(cmd)
+	AddRunDeployFlags(cmd)
```

Comment:
`tail` flag defaults to `false`. I think it should default to `true` in dev mode.